storage/redis_cluster.go

```diff
@@ -80,44 +82,55 @@ func WaitConnect(ctx context.Context) bool {
 	}
 }
 
-func singleton(cache bool) redis.UniversalClient {
+func singleton(cache, analytics bool) redis.UniversalClient {
```

Comment:
Are you sure we need boolean flag here?
How separate cache storage works then? Does it mean it currently broken?

Reply:
@buger what do you mean? the cache flag is also passed to singleton func.

Reply:
Damn not sure how I read that. Indeed.